documentation/tutorials/get-started-with-phoenix.md

```diff
@@ -0,0 +1,506 @@
+
+It also tells where to find the resources within this API - in the `MyAshPhoenixApp.Blog.Registry` which we will make now.
+
+### Create Registry
+
+```elixir
+# lib/my_ash_phoenix_app/blog/registry.ex
+
+defmodule MyAshPhoenixApp.Blog.Registry do
+  use Ash.Registry,
+    extensions: [
+      # This extension adds helpful compile time validations
+      Ash.Registry.ResourceValidations
+    ]
+
+  entries do
+    entry MyAshPhoenixApp.Blog.Post
+  end
+end
+```
+
+In the registry we enumerate the Ash resources we want our Ash API to contain. In this case its just one resource called `Post`.
+
+### Creating the `Post` Resource
+
+Its convention to place all the resource in their own resources folder. So when we create `Post` we will place it in `lib/my_ash_phoenix_project/blog/resources/post.ex`. So the structure after making the resource should look like so:
+
+```
+lib/
+├─ my_ash_phoenix_app/
+│  ├─ blog/
+│  │  ├─ registry.ex
+│  │  ├─ resources/
+│  │  │  ├─ post.ex
+│  ├─ blog.ex
+```
+
+Below is resource module. Read the comments carefully, every line is explained.
+
+```elixir
+defmodule MyAshPhoenixApp.Blog.Post do
+  # Using Ash.Resource here turns this model into an Ash resource.
+  use Ash.Resource,
+    # Tells Ash you want this resource to store its data in postgres.
+    data_layer: AshPostgres.DataLayer
+
+  # The postgres keyword is specific to the AshPostgres module.
+  # It tells postgres what to call the table and to
+  # communicate with postgres through MyPhoenixApp.Repo
+  postgres do
+    # Tells postgres what to call the table
+    table "posts"
+    # Tells Ash how to interface with the postgres table
+    repo MyAshPhoenixApp.Repo
+  end
+
+  actions do
+    # Exposes default built in actions to modify the resource
+    defaults [:create, :read, :update, :destroy]
+  end
+
+  # Attributes are the simple pieces of data that exist on your resource
+  attributes do
+    # Add an autogenerated UUID primary key called `:id`.
+    uuid_primary_key :id
+    # Add a string type attribute called `:title`
+    attribute :title, :string do
+      # We don't want the title to ever be `nil`
+      allow_nil? false
+    end
+
+    # Add a string type attribute called `:content`
+    # If not specified content can be `nil`
+    attribute :content, :string
+  end
+end
+```
+
+### Migrate Database
+
+We have specified the resource attributes and actions in Ash. But we have yet to create them in our data layer (in our case postgres).
+
+We created our database earlier, but now we need to populate it. We do this by generating and performing a migration.
+
+We can use a generator to produce a migration for us. Ash can deduce what needs to go into the migration and do the hard work for us, just use the command below.
+
+```bash
+> mix ash_postgres.generate_migrations --name initial_migration
```

Comment:
Not super necessary but may want to include something about how the --name flag is optional, but recommended (especially when working with other devs)

Reply:
I don't think I want people to know the `--name` flag is optional. 
If they know its optional there'll be someone who'll decide to not use it. 😅